File: iniciante/leetcode/remove_duplicates_from_sorted_array/removeDuplicates.py

def removeDuplicates(nums):
    
    l = 0
    r = 0

    while r < len(nums):

        if nums[l] != nums[r]:
            l+=1
            nums[l] = nums[r]
        
        r+=1

    return l+1, nums




print(removeDuplicates([0,0,1,1,1,2,2,3,3,4]))

File: iniciante/leetcode/remove_duplicates_from_sorted_array/removeDuplicates.js

var removeDuplicates = function(nums) {
    
    let left = 0

    for (let right = 0; right < nums.length; right++) {
        const element = nums[right];

        if (element != nums[left]){
            left++
            nums[left] = nums[right]
        }
        
    }

    return left+1
};


console.log(removeDuplicates([1,1,2]))